modules/ROOT/pages/mule-app-strategies.adoc

```diff
@@ -6,14 +6,16 @@ endif::[]
 
 These topics cover Mule app configurations that you can use with various Mule components:
 
-* xref:target-variables.adoc[Enrich Data with Target Parameters]
-* xref:error-handling.adoc[Error Handling]
- ** xref:mule-error-concept.adoc[Mule Errors]
 * xref:configuring-properties.adoc[Configure Properties]
  ** xref:mule-app-properties-to-configure.adoc[Configure Property Placeholders for Mule Apps]
  ** xref:mule-app-properties-system.adoc[System Properties]
+ ** xref:custom-configuration-properties-provider.adoc[Custom Configuration Properties Provider]
+* xref:error-handling.adoc[Error Handling]
+  ** xref:mule-error-concept.adoc[Mule Errors]
+* xref:streaming-about.adoc[Streaming in Mule Apps]
+ ** xref:streaming-strategies-reference.adoc[Streaming Strategies Reference]
+* xref:shared-resources.adoc[Shared Resources]
 * xref:reconnection-strategy-about.adoc[Reconnection Strategies]
  ** xref:reconnection-strategy-reference.adoc[Reconnection Strategy Reference]
 * xref:sources-reference.adoc[Sources Reference]
-* xref:streaming-about.adoc[Streaming in Mule Apps]
- ** xref:streaming-strategies-reference.adoc[Streaming Strategies Reference]
+* xref:target-variables.adoc[Enrich Data with Target Parameters]
```

Comment:
Shoot, I left out target-variables from the rankings: It should go right above reconnection-strategy links.